skbio/metadata/_enan.py

```diff
@@ -0,0 +1,87 @@
+# ----------------------------------------------------------------------------
```

Comment:
Would it be possible to add unit tests specific to the methods in this file?

Reply:
...I think they're here?

Reply:
we should also include tests for missing.py https://github.com/qiime2/qiime2/blob/dev/qiime2/core/tests/test_missing.py